src/index.js

```diff
@@ -74,6 +75,11 @@ function validateOptions(options) {
       throwInvalid(`Invalid option — ${option} must be an integer.`);
     }
   });
+
+  // Invalid cache dir (not an absolute path)
+  if ('cacheDir' in options && !path.isAbsolute(options.cacheDir)) {
```

Comment:
Out of curiosity, why does it have to be an absolute path?

Reply:
It was mostly an attempt at making things more explicit; a relative path could be confusing depending on your build setup. But the more I think about it, the less it makes sense; developers can still use absolute paths if they want, and if they want to use relative ones it's on them to figure out what the base dir is.

I'll remove this check and just use a `path.parse` to do basic validation on the path.